src/language/walker/syntaxWalker.ts

```diff
@@ -253,6 +265,15 @@ module Lint {
                     this.visitFunctionDeclaration(<ts.FunctionDeclaration> node);
                     break;
 
+                case ts.SyntaxKind.FunctionExpression:
+                    this.visitFunctionExpression(<ts.FunctionExpression> node);
+                    break;
+
+                case ts.SyntaxKind.GetAccessor:
```

Comment:
ok, I'll split them out.

it sort of pains me to have to write out the following:
```ts
    public visitFunctionDeclaration(node: ts.FunctionDeclaration) {
        this.handleCallSignature(node);
        super.visitFunctionDeclaration(node);
    }

    public visitGetAccessor(node: ts.AccessorDeclaration) {
        this.handleCallSignature(node);
        super.visitGetAccessor(node);
    }

    public visitMethodDeclaration(node: ts.MethodDeclaration) {
        this.handleCallSignature(node);
        super.visitMethodDeclaration(node);
    }

    // ... 2 more identical functions
```

but I can't think of anything better given the current architecture...